core/src/main/java/cn/edu/tsinghua/iginx/engine/shared/operator/type/OperatorType.java

```diff
@@ -55,7 +56,7 @@ public static boolean isBinaryOperator(OperatorType op) {
     }
 
     public static boolean isUnaryOperator(OperatorType op) {
-        return op == Project || op == Select || op == Sort || op == Limit || op == Downsample || op == RowTransform || op == SetTransform || op == MappingTransform || op == Delete || op == Insert || op == Rename || op == Reorder;
+        return op == Project || op == Select || op == Sort || op == Limit || op == Downsample || op == RowTransform || op == SetTransform || op == MappingTransform || op == Delete || op == Insert || op == Rename || op == Reorder || op == AddSchemaPrefix;
```

Comment:
Highly ugly implementation. Should be restructured from last time.

Reply:
fixed